test/integration/root_overview_test.rb

```diff
@@ -147,6 +147,18 @@ class RootOverviewTest < ActionDispatch::IntegrationTest
     assert page.has_css?("#publication-list-container table tbody tr:nth-child(3) td:nth-child(5)", text: "4 days")
   end
 
+  test "Shows link to Collections Publisher when reviewing in review documents" do
+    FactoryBot.create(:user)
+    FactoryBot.create(:guide_edition, title: "XXX", slug: "xxx",
+                       state: "in_review", review_requested_at: 4.days.ago)
```

Comment:
I'm not sure why this is needed - don't we want to show the link regardless of whether there are any mainstream items in review?

Reply:
Good point, I've removed this line